.scripts/pre-push

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+
+# If the publication-model has updates that change the model structure,
+# the model version should be updated.
+
+TARGET="publication-model"
+BUILD_FILE="${TARGET}/build.gradle"
+PATTERN="^def modelVersion = '"
+MODEL_CHANGE_MESSAGE="Changes in the $TARGET package, which imply\
```

Comment:
This message contains what to do if you have changes that don't require a model-version update.